src/modules/time.rs

```diff
@@ -0,0 +1,29 @@
+use ansi_term::Color;
+use chrono::{DateTime, Local};
+
+use super::{Context, Module};
+
+/// Outputs the current time
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("time")?;
+
+    // Remove when logic for disabled by default exists
+    if module.config_value_bool("disabled").unwrap_or(true) {
+        return None;
+    }
+
+    let is_12hr = module.config_value_bool("12hr").unwrap_or(false);
+    let default_format = if is_12hr { "%I:%M%p" } else { "%H:%M" };
```

Comment:
It looks like there are premade format strings for exactly that:
```suggestion
    let default_format = if is_12hr { "%r" } else { "%T" };
```

Reply:
Done! (https://github.com/starship/starship/pull/138/commits/6d9f8eaface17cf71901a74ab0679cb719ff98f7)